src/System.Text.Encoding.CodePages/src/System/Text/SBCSCodePageEncoding.cs

```diff
@@ -70,9 +91,10 @@ protected override unsafe void LoadManagedCodePage()
                 // Get our mapped section 65536 bytes for unicode->bytes, 256 * 2 bytes for bytes->unicode
                 // Plus 4 byte to remember CP # when done loading it. (Don't want to get IA64 or anything out of alignment)
                 byte* pNativeMemory = GetNativeMemory(65536 * 1 + 256 * 2 + 4 + iExtraBytes);
```

Comment:
I suggest to move the number of bytes we calculate to some variable to avoid re-writing the whole calculation